SolidBeginnerWorkshopNotes.md

```diff
@@ -0,0 +1,149 @@
+    <https://solidproject.org/TR/protocol>
+    - *The specifications in the ecosystem describe how Solid servers
+        and clients can be interoperable by using Web communication
+        protocols, global identifiers, authentication and authorization
+        mechanisms, data formats and shapes, and query interfaces.*
+
+- **Solid Pod: Pods are where you store your data**.
+    - Any kind of data can be stored in a Solid Pod
+        - Structured data, files
+    - Linked Data which allows different applications to work with the
+        same data
+        - *Linked Data is a set of design principles for sharing
+            machine-readable interlinked data on the Web.*
+    - Once data is in your Pod, you control access to it.
+
+- **Pod Provider:**
+    - You can get a Pod from a Pod Provider, or you can self-host your
+        Pod. (You can also implement a new Pod Provider if you like
+        implementing from specifications!)
+    - <https://solidproject.org/users/get-a-pod>
+        - Many Pod Providers to choose from! Some community-based,
+            some by companies.
+        - A Pod Provider will usually advertise the underlying cloud
+            infrastructure it is using, and the hosting location. For
+            example, Inrupt Pod Spaces uses Amazon for hosting and the
+            hosting location is Germany.
+
+- **Solid applications:**
+    - Any type of application can be built using the Solid Protocol.
+    - Applications can make requests to retrieve data from your Pod (via your Pod Provider), and also to store data on your Pod. This is the essential part of how Solid works. 
+        - Keep in mind that you ALWAYS control access of your files on your Pod, so you can revoke access to an application at any time. 
+    - Example of applications: <https://solidproject.org/apps>
+    - Example active funded projects:
+        - health care use case (name?) by Marc Haddle
+        - ???
+
+## Solid in context
+
+- Solid is a specification consisting of several Technical Reports,
+    each edited by a Solid panel.
+    - Anyone can contribute by [filing an issue on
+        GitHub](https://github.com/solid/specification/issues)
+    - Technical Reports: <https://solidproject.org/TR/> include the
+        [Solid Protocol](https://solidproject.org/TR/protocol), [Solid
+        WebID Profile](https://solid.github.io/webid-profile/), [Web
+        Access Control](https://solidproject.org/TR/wac), [Shape
+        Trees](https://shapetrees.org/TR/specification/), [Solid
+        Chat](https://solid.github.io/chat/), and lots more!
+
+- The authors of the technical reports are the [Solid Community
+    Group](https://www.w3.org/groups/cg/solid/) (CG) of the
+    [W3C](https://www.w3.org/).
+    - W3C is the World Wide Web Consortium: they *develop standards
+        and guidelines to help everyone build a web based on the
+        principles of accessibility, internationalization, privacy and
+        security.*
```

Comment:
There is also now a working group (to make the standard oficially W3C endorsed), under the name Linked Web Storage https://www.w3.org/groups/wg/lws/